awscli/customizations/logs/tail.py

```diff
@@ -128,7 +152,7 @@ class TailCommand(BasicCommand):
         {
             'name': 'format',
             'default': 'detailed',
-            'choices': ['detailed', 'short'],
+            'choices': ['detailed', 'short', 'message-only', 'json'],
```

Comment:
I think we need to remove the `message-only` option from the choices. Looks like an update left over from the previous version